website/views.py

```diff
@@ -307,6 +307,10 @@ def resolve_guid(guid, suffix=None):
 
             return send_from_directory(preprints_dir, 'index.html')
 
+        # Handle DraftNodes - these should never be accessed directly
+        if isinstance(referent, DraftNode):
+            raise HTTPError(http.NOT_FOUND)
```

Comment:
What other handling do we need to add to keep DraftNodes from being accessed? I've tried to exclude them from search and they are not returned under Node endpoints.